### PR TITLE
FLPROD-796: Extend snippet interface with DELETE and GET a single sni…

### PR DESCRIPTION
**Pull request by WowVeryLogin**
Adding missing method to Get and Delete a single snippet

### CODE REVIEW

File: snippets.go [hunk 1]
```diff
@@ -33,6 +38,48 @@ type SnippetRequest struct {
...
+	if err != nil {
+		return err
+	}
+
+	result := SnippetResponse{}
+	if err := json.Unmarshal(res, &result); err != nil {
+		return fmt.Errorf("%s: %w", errUnmarshalError, err)
+	}
+
+	if result.Success != true {
```
Comment (jacobbednarz) [inline snippets.go]:
```suggestion
	if !result.Success {
```

File: snippets.go [hunk 2]
```diff
@@ -33,6 +38,48 @@ type SnippetRequest struct {
...
+		return fmt.Errorf(result.Messages[0].Message)
```
Comment (jacobbednarz) [inline snippets.go]:
```suggestion
		return errors.New(result.Messages[0].Message)
```

File: snippets.go (hunk 1 above)
Comment (WowVeryLogin) [reply snippets.go]: Fixed

File: snippets.go (hunk 2 above)
Comment (WowVeryLogin) [reply snippets.go]: Fixed

**Pull request merged**